scripts/build-windows.bat

```diff
@@ -34,7 +34,7 @@ cmake --build %BUILD_DIR% --verbose --parallel --config %BUILD_CONFIGURATION% ||
 
 if "%INSTALL%" == "ON" (
     @echo Installing...
-    cmake --install %BUILD_DIR% || exit /b 1
+    cmake --install %BUILD_DIR% --config %BUILD_CONFIGURATION% || exit /b 1
```

Comment:
How was the previous version without this flag working?

Reply:
When no --config is given, it installs all existing configs. But since do-all-windows.bat sets `-DCMAKE_CONFIGURATION_TYPES=%BUILD_CONFIG%`, only the %BUILD_CONFIG% exists and it gets installed.

I think it would be better to put the -DCMAKE_CONFIGURATION_TYPE option in this file instead of setting it from the do-all script. I will do that. Also, I realized the CMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%... line above in line 27 which is not used by the GA workflow. Setting that from the do-all script is more appropriate, I think.